doc/api/os.md

```diff
@@ -19,9 +19,19 @@ system.
 added: v0.5.0
 -->
 
+    Stability: 0 - Deprecated: Use [`process.arch`][] instead.
+
 Returns the operating system CPU architecture. Possible values are `'x64'`,
 `'arm'` and `'ia32'`. Returns the value of [`process.arch`][].
 
+## os.BYTEORDER
```

Comment:
@vkurchatkin ... considered those but given that they do not have equivalent `process.*` properties, I'd leave those alone for now. Those could easily be turned into properties too tho and would have no problem doing so.